ingest/ingest-publications.py

```diff
@@ -390,7 +391,39 @@ def generate(threads):
     g1 = g1 + describe(sparqlendpoint,get_pub_query)
     print("EMAIL: ", EMAIL)
 
+# section to create chunked files
+    chunk = 500
```

Comment:
500 is a good value to use? Does it need to be configurable?

Reply:
Seems like this is the core of the chunking process? Does it make sense to have a utility handler class that takes care of the chunking? Seems like this might be a more generic issue we may encounter with other AWS integrations...

Reply:
500 was a number where the files easily fit in the AWS 10mb boundary, I could increase the size. I could also parameterize this but decided to leave it in the code. I didn't look up any canned methods for this, just rolled my own

Reply:
this is a parameter now